packages/twenty-website/src/app/_components/ui/layout/header/index.tsx

```diff
@@ -1,12 +1,19 @@
-'use client';
+import { desc } from 'drizzle-orm';
 
 import { HeaderDesktop } from '@/app/_components/ui/layout/header/HeaderDesktop';
 import { HeaderMobile } from '@/app/_components/ui/layout/header/HeaderMobile';
+import { findOne } from '@/database/database';
+import { githubStarsModel } from '@/database/model';
+
+export const AppHeader = async () => {
+  const githubStars = await findOne(
+    githubStarsModel,
+    desc(githubStarsModel.id),
+  );
 
-export const AppHeader = () => {
   return (
     <>
-      <HeaderDesktop />
+      <HeaderDesktop numberOfStars={githubStars?.[0]?.numberOfStars} />
```

Comment:
Can you expose the formatted number of stars as an endpoint for Thomas to retrieve on Framer?